src/routes/DigitalAssets/routes/AddAsset/AddAssetView.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import React from 'react'
+
+import {
+  CloseableScreen,
+  DigitalAssetEditForm,
+} from 'components'
+
+type Props = {
+  openView: () => void,
+  closeView: () => void,
+  closeClick: () => void,
+} & React$ElementConfig<typeof DigitalAssetEditForm>
+
+const AddAssetView = ({
```

Comment:
I suggest to name route `add`
so full path would be `digital-assets/add`
View in this case should be named DigitalAssetsAddView